Makefile

```diff
@@ -0,0 +1,17 @@
+all: push
+
+# 0.0 shouldn't clobber any release builds
+TAG = 0.0
+PREFIX = kobolog/gorb
+
+binary: main.go
+	CGO_ENABLED=0 GOOS=linux godep go build -a -installsuffix cgo -ldflags '-w' -o docker/gorb ./main.go ./http.go
```

Comment:
If you remove both `./main.go` and `./http.go` then it will just compile
the whole package (but not `gorb-docker-link`, since it's another `main`
package).

On Tue, Dec 8, 2015 at 10:59 PM, Manuel Alejandro de Brito Fontes <
notifications@github.com> wrote:

> In Makefile <https://github.com/kobolog/gorb/pull/12#discussion_r47049124>
> :
>
> > @@ -0,0 +1,17 @@
> > +all: push
> > +
> > +# 0.0 shouldn't clobber any release builds
> > +TAG = 0.0
> > +PREFIX = kobolog/gorb
> > +
> > +binary: main.go
> > +	CGO_ENABLED=0 GOOS=linux godep go build -a -installsuffix cgo -ldflags '-w' -o docker/gorb ./main.go ./http.go
>
> Why -installsuffix cgo
>
> Removed.
>
> Also, I think it will build just fine without listing the source files in
> the end.
>
> Without ./http.go I see this error
>
> old-mbp:gorb aledbf$ make
> CGO_ENABLED=0 GOOS=linux godep go build -a -ldflags '-w' -o docker/gorb ./main.go
> # command-line-arguments
> ./main.go:74: undefined: serviceCreateHandler
> ./main.go:75: undefined: backendCreateHandler
> ./main.go:76: undefined: backendUpdateHandler
> ./main.go:77: undefined: serviceRemoveHandler
> ./main.go:78: undefined: backendRemoveHandler
> ./main.go:79: undefined: serviceStatusHandler
> ./main.go:80: undefined: backendStatusHandler
> godep: go exit status 2
> make: *** [binary] Error 1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kobolog/gorb/pull/12/files#r47049124>.
>